Mathlib/Algebra/Algebra/Subalgebra/IsSimpleOrder.lean

```diff
@@ -0,0 +1,33 @@
+/-
+Copyright (c) 2020 Kenny Lau. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kenny Lau
+-/
+import Mathlib.LinearAlgebra.FiniteDimensional.Defs
+import Mathlib.LinearAlgebra.Dimension.FreeAndStrongRankCondition
+import Mathlib.LinearAlgebra.FreeModule.StrongRankCondition
+
+/-!
+If `A` is a domain, and a finite-dimensional algebra over a field `F`, with prime dimension,
+then there are no non-trivial `F`-subalgebras.
+-/
+
+open FiniteDimensional Submodule
+
+theorem Subalgebra.isSimpleOrder_of_finrank_prime (F A) [Field F] [Ring A] [IsDomain A]
```

Comment:
I think you renamed this from `FiniteDimensional.Subalgebra.isSimpleOrder_of_finrank_prime`

Reply:
I'm pretty sure that was a mistake? I didn't add a deprecation as it is unused in Mathlib.